scripts/Minify-Scripts.ps1

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env powershell
+#requires -version 4
+
+[CmdletBinding(PositionalBinding = $false)]
+param()
+
+Set-StrictMode -Version 2
+$ErrorActionPreference = 'Stop'
+
+npm install -g uglifycss
```

Comment:
Can you add a package.json file and install this into a local scope? Installing globally affects the machine state